### PR TITLE
Simplify notification types

### PR DESCRIPTION
**Pull request by hpeebles**

### CODE REVIEW

File: backend/canisters/community/impl/src/updates/add_members_to_channel.rs [hunk 1]
```diff
@@ -185,14 +185,15 @@ fn commit(
...
-        let notification = Notification::AddedToChannelNotification(AddedToChannelNotification {
+        let notification = Notification::AddedToChannel(AddedToChannelNotification {
             community_id: state.env.canister_id().into(),
             community_name: state.data.name.clone(),
             channel_id,
             channel_name: channel.chat.name.clone(),
             added_by,
             added_by_name,
-            timestamp: now,
+            community_avatar_id: state.data.avatar.as_ref().map(|d| d.id),
```
Comment (megrogan) [inline backend/canisters/community/impl/src/updates/add_members_to_channel.rs:195]: Are these supposed to be the same?

Comment (hpeebles) [reply backend/canisters/community/impl/src/updates/add_members_to_channel.rs:195]: No! Fixed

File: frontend/openchat-client/src/openchat.ts [hunk 2]
```diff
@@ -3088,69 +3083,73 @@ export class OpenChat extends OpenChatAgentWorker {
...
         }
 
-        const serverChat = this._liveState.serverChatSummaries.get(chatId);
+        const minVisibleEventIndex = serverChat.kind === "direct_chat" ? 0 : serverChat.minVisibleEventIndex;
+        const latestClientEventIndex = Math.max(eventIndex, serverChat.latestEventIndex);
 
-        if (serverChat === undefined || serverChat.latestEventIndex >= message.index) {
-            return;
+        if (isReaction) {
+            // TODO first clear the existing cache entry
```
Comment (megrogan) [inline frontend/openchat-client/src/openchat.ts:3139]: Is this not really important?

Comment (hpeebles) [reply frontend/openchat-client/src/openchat.ts:3139]: Its not urgent but I left the TODO just for future reference

**Review by megrogan — APPROVED** [review]

**Pull request merged**